lib/solarwinds_apm/patch/otel_registry.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module SolarWindsAPM
+  module Patch
+    module OtelRegistry
+      def install_instrumentation(instrumentation, config)
+        if !instrumentation.present?
+          ::OpenTelemetry.logger.debug "Instrumentation: #{instrumentation.name} skipping install given corresponding dependency not found"
+        elsif instrumentation.install(config)
+          ::OpenTelemetry.logger.info "Instrumentation: #{instrumentation.name} was successfully installed with the following options #{instrumentation.config}"
+        else
+          ::OpenTelemetry.logger.warn "Instrumentation: #{instrumentation.name} failed to install"
+        end
+      rescue => e # rubocop:disable Style/RescueStandardError
+        ::OpenTelemetry.handle_error(exception: e, message: "Instrumentation: #{instrumentation.name} unhandled exception during install: #{e.backtrace}")
+      end
+    end
+  end
+end
```

Comment:
Since https://github.com/open-telemetry/opentelemetry-ruby/releases/tag/opentelemetry-registry/v0.3.1 is now released, do we still need this patch?

Reply:
I'd like to keep it here as in the future, we may have more similar case.

Reply:
From quick test no change needed to use updated registry, a new install of `solarwinds_apm` will automatically install it. @xuan-cao-swi can put in a dummy patch to keep the mechanism in place.